bmc/src/main/java/foundation/icon/btp/bmc/ICONSpecific.java

```diff
@@ -148,4 +148,16 @@ public interface ICONSpecific {
      */
     @External(readonly = true)
     BigInteger getMode();
+
+    /**
+     * (EventLog) Logs the event that handle the relay message
+     * The tracker monitors this event.
+     * <p>
+     * indexed: 1
+     *
+     * @param _prev  String ( BTP Address of the previous BMC )
+     * @param _count Integer ( number of the processed BTP message )
+     */
+    @EventLog(indexed = 1)
```

Comment:
No needs to be indexed. It's better to use it as data for identifying the previous network
```suggestion
    @EventLog(indexed = 0)
```